### ISSUE TITLE
[JOSS Review] Consider more verbose function names

### ISSUE THREAD

**Issue by richardsc** [tbep-tech/tbeptools#47]
This comment is really about coding style, and while I hope the authors consider it seriously I don't consider this to be a blocking issue. I recognize that changing function names is a major breaking change, especially for a package that is as fully-developed as this one, and likely has a long history of use already within the institutions for which it was designed.

I like the consistency in the function verb prefixes (e.g. `read_`, `show_`, `anlz_`) however as an unfamiliar user I find it hard to parse the spectific intentions of the functions from the suffixes. 

In general, I would prefer to see fully descriptive function names -- e.g. `analyze_` instead of `anlz_`, `water_quality` instead of `wq`, etc. As an example `anlz_avedat()` is completely opague to me as a function name -- the documentation says that it "Estimates annual means", so why not call it `analyze_annual_means()`. The argument against this is that it's a lot of typing, but in this day and age of good IDEs and code completion I prefer real words instead of confusing abbreviations.

Another thing to consider is that the mix of `snake_case` for the prefixes and *no* case convention for the suffixes is also hard to parse. I think that much of the functionality of the package would be more easily deduced if the functions were named as above, e.g. `analyze_annual_means()` or `analyze_site_attainments()`.

(Linking openjournals/joss-reviews#3485 )

**Comment by fawda123** [comment]
Thanks @richardsc.  I fully agree that function names should be as intuitive as possible.  Since hindsight is of course 20/20, I realize I could have done a better job for the function names in tbeptools.  I did consider whether changing the names was practical, but as it stands, this package is used across several dozen repositories on our group web page (https://github.com/tbep-tech) and in CI actions through GitHub Actions.  

Rather than risk breaking lots of time-sensitive things (some of which are under regulatory compliance, e.g., https://tbep-tech.github.io/tbnmc-compliance-assessment-2020/), I figured a good compromise was to provide some descriptive text on the web page and to point the reader there in the manuscript.  I hope that's a good compromise.  Again, I appreciate your comments on the package and paper.  I'm sure we've got a better product as a result of these review!
